### PR TITLE
Config to external file

### PR DESCRIPTION
**Pull request by adriacidre**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+global.config = require('./config.json');
```
Comment (repejota) [inline app.js]: Just var config = require('./config.json'); is enough

Comment (repejota) [inline app.js]: var config = require('./config.json'); is enough

Comment (adriacidre) [inline app.js]: done

On Mon, Aug 3, 2015 at 5:53 PM, Raül Pérez notifications@github.com wrote:

> In app.js
> https://github.com/repejota/nats-mon/pull/11#discussion_r36099172:
> 
> > @@ -1,3 +1,5 @@
> > +global.config = require('./config.json');
> 
> var config = require('./config.json'); is enough
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/repejota/nats-mon/pull/11/files#r36099172.

## 

## _Adrià Cidre_

E-Mail adria.cidre@gmail.com - Web Page http://en.blog.oridoki.com/ -
Linkedin http://www.linkedin.com/in/adriacidre
Skype: _kumulo_
Twitter: @adriacidre http://twitter.com/adriacidre

**Comment by repejota** [comment]
Thanks!

**Pull request merged**